nms/packages/magmalte/app/views/subscriber/SubscriberUtils.js

```diff
@@ -36,6 +36,8 @@ export function convertBitToMbit(val: number) {
   return (val / mBIT).toFixed(2);
 }
 
+export const NETWORK_TYPES = ['NT_EPC', 'NT_5GC'];
```

Comment:
Use `Object.freeze`

```suggestion
export const NetworkTypes = Object.freeze({
  EPC: 'NT_EPC',
  5GC: 'NT_5GC',
});
```

If you need the array `['NT_EPC', 'NT_5GC']` you can use `Object.values(NetworkTypes)`

Replace the raw occurrences of the string `'NT_EPC'` with `NetworkTypes.EPC`, and something similar with `'NT_5GC'`.

Reply:
On second thought - rename this to `CoreNetworkTypes`, and have all references to these values be `coreNetworkType`, or something similar. We're already using *'network type'* to refer to something else.

Reply:
I changed it to `CoreNetworkTypes`.